deployer/src/deployer/upload.py

```diff
@@ -119,7 +118,7 @@ class UploadFileTask(UploadTask):
     Class for file upload tasks.
     """
 
-    def __init__(self, file_path, key):
+    def __init__(self, file_path: Path, key: str):
```

Comment:
This little change has no runtime effect. But if you do it, when you're further down in the code and type:
```
if self.file_path.
```
a VSCode autocomplete thing pops up and it now knows that `self.file_path` is of type `pathlib.Path` so it knew to immediately auto-suggest `parent` after I just pressed `p`.